Avoid an assertion failure and crash of the python interpreter caused by
pygtk's usage of the deprecated PyCObject and Python 2.7.

https://bugzilla.redhat.com/show_bug.cgi?id=621887

Index: gajim-0.13.90/src/gajim.py
===================================================================
--- gajim-0.13.90.orig/src/gajim.py
+++ gajim-0.13.90/src/gajim.py
@@ -133,7 +133,7 @@ if os.name == 'nt':
     sys.stderr = MyStderr()
 
 # PyGTK2.10+ only throws a warning
-warnings.filterwarnings('error', module='gtk')
+warnings.filterwarnings('error', module='gtk', append=True)
 try:
     import gobject
     import gtk
